tap_exacttarget/endpoints/data_extensions.py

```diff
@@ -60,7 +60,9 @@ def _get_extensions(self):
             'DataExtension',
             FuelSDK.ET_DataExtension,
             self.auth_stub,
-            props=['CustomerKey', 'Name'])
+            props=['CustomerKey', 'Name'],
+            batch_size=self.config.get('batch_size', 2500)
```

Comment:
Due to some requirements on the Stitch side, could we convert what we get from the config into an integer? ie `int(self.config.get('batch_size', 2500)`

Reply:
sure @KAllan357 - happy to make that change

Reply:
I just pushed a commit which addresses this feedback